public/app/plugins/datasource/loki/components/LokiQueryEditor.tsx

```diff
@@ -172,6 +172,7 @@ export const LokiQueryEditor = React.memo<LokiQueryEditorProps>((props) => {
           app={app}
           maxLines={datasource.maxLines}
           datasource={datasource}
+          experimentalChunkRange={query.experimentalChunkRange}
```

Comment:
```suggestion
          chunkRange={query.chunkRange}
```

Reply:
I am a bit hesitant to change the variable name, as those are persisted to the dashboard JSON, when you save a panel on a dashboard. So if people now save dashboards with chunking enabled, it might be hard to import those dashboard if we decide to change that property in the future. 

@gabor WDYT?

Reply:
yeah, that's a legitimate problem.

some options:
1. `chunkRangeExperimental` ? (having `experimental` at the end sounds a little better for me)
2. `testing` or `test` or `trial` instead of `experimental`?
3. keep the current name and add a comment describing the persistence-problem?
4. for the initial run allow changing only in explore-mode, and we can delay this discussion to the future allow-in-dashboard-step 😁 

(in general, would be great to be able to define functions how a `LokiQuery` gets stored and gets loaded back in.. but i don't think we have that currently)

Reply:
We could also go with an `experimental` struct inside `LokiQuery`. Something like

```
  experimental: {
    chunkRange: string;
  }
```

Reply:
The dashboard persistence is a good reason, but again: what would be the difference between having an attribute called `experimentalChunk` or any other name that you think it could change in the future? 
Either if it's called `experimentalSomething` or just `something`, the problem is that it changes, not the experimental part, unless I'm missing anything.

What I don't like about having `experimentalVariable` all over the implementation is that as an internal user it's not giving me any useful information. In `querySplitting.ts` it's the chunk size and has nothing experimental about it. The information seems relevant only for the external user.

Reply:
Your points are understandable.
Just regarding
> TL;DR the name of the variable doesn't fix the problem of an imported/exported dashboard with an attribute that later changed.
If we now use something like `experimentalChunkRange` with `string` values, but decide to use `number` with our "non-experimental" version, we won't get any problems importing dashboards.